stroom-app/Dockerfile

```diff
@@ -38,9 +38,10 @@ WORKDIR /stroom
 
 #Copy the content packs in so they get imported on first run of Stroom
 #TODO this should be a singley named jar
-ADD ./build/libs/stroom-app-all.jar /stroom/stroom-app-all.jar
-ADD ./prod.yml /stroom/config.yml
-ADD ./docker-entrypoint.sh /usr/local/bin/
```

Comment:
For Docker to be able to see files outside of the immediate stroom-app, I had to effectively point at the parent project, this is why the relative paths have changed.